common/src/chain/transaction/utxo_outpoint.rs

```diff
@@ -17,19 +17,7 @@ use crate::chain::{transaction::Transaction, Block, GenBlock, Genesis};
 use crate::primitives::Id;
 use serialization::{Decode, Encode};
 
-#[derive(
-    Debug,
-    Clone,
-    PartialEq,
-    Eq,
-    Encode,
-    Decode,
-    Ord,
-    PartialOrd,
-    serde::Serialize,
-    serde::Deserialize,
-    rpc_description::HasValueHint,
-)]
+#[derive(Debug, Clone, PartialEq, Eq, Encode, Decode, Ord, PartialOrd)]
```

Comment:
Some times I think we should just make a static check that prevent Encode/Decode and serde::Serialize/Deserialize to coexist.